### ISSUE TITLE
Update Sublime Text syntax highlighting example

### ISSUE THREAD

**Issue by Thom1729** [kireerik/refo#41]
The [Examples page](https://github.com/kireerik/refo/tree/main/example#sublime-text) recommends the following configuration for JS Custom:

```json
{
  "configurations": {
    "Default": {
      "custom_template_tags": {
        "html": "scope:text.html.basic"
        , "": "scope:text.html.basic"
      }
    }
  }
}
```

In current versions of JS Custom, custom template options have been expanded. The `custom_template_tags` option has been deprecated in favor of a `custom_templates` option with various sub-options:

```json
{
  "configurations": {
    "Default": {
      "custom_templates": {
        "default": "scope:text.html.basic",
        "tags": {
          "html": "scope:text.html.basic"
        }
      }
    }
  }
}
```

The old suggested configuration using `custom_template_tags` should still work, but as I recall, using `""` as a custom tag was buggy, which is why the new `default` sub-option exists.

**Comment by kireerik** [comment]
Yes, both works without bugs.

**Comment by kireerik** [comment]
Currently I am working on replacing [refo](https://github.com/kireerik/refo) by using [Astro](https://astro.build/). The progress looks good so far. So my current plan with [refo](https://github.com/kireerik/refo) is that once I am finished with the replacement I will update this repository as well and my current plan is that [refo](https://github.com/kireerik/refo) will become a static site and resume example (instead of a build tool).

> Somewhat recently I started to simplify (#40) how [refo](https://github.com/kireerik/refo) would work. However during working on this I came across [Astro](https://astro.build/). With [Astro](https://astro.build/) it seems like it is possible to implement all the same functionalities so I can eliminate the need for the current [refo](https://github.com/kireerik/refo) and just use [Astro](https://astro.build/). This results in less effort from my side since I don't need to simplify this separate build tool.
When I created [refo](https://github.com/kireerik/refo) I couldn't find any alternative, but now I found [Astro](https://astro.build/) (It didn't exist at that time.).

**Comment by kireerik** [comment]
Also once we will have this new setup (using [Astro](https://astro.build/) (and likely [Emotion](https://emotion.sh/docs/introduction) as well (during generation (and not on the frontend)))) this new setup will be a really strong setup to build (optimized (all output is optimized and minified)) static sites (and maybe also have non static parts as well as and if needed) with (fully) reusable parts.

&nbsp;
Sometimes I am thinking about how can we monetize this setup (Which I think will be a really good one.) and preferabbly build a business and or businesess relying on it.

So far I have no groundbraking ideas:
- offer and implement this setup for website builders like @webflow for example (and later maybe for others as well)
- use the setup in an agency creating static sites for companies
- build news websites to medium sized news websites (and later for the biggest as well)
- create a course on how to create a static site and resume using this simple setup
- just get sponsors for the setup

For monetization and or building a business and or businesses I am definitely looking for partners. If we have a good idea and approach that we can see will be a really good business we can start to collaborate and work on it.

**Comment by kireerik** [comment]
closed by 0dc47f8
